[2023-07-02T08:54:16.481+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2023-07-02T08:24:29.703225+00:00 [queued]>
[2023-07-02T08:54:16.488+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2023-07-02T08:24:29.703225+00:00 [queued]>
[2023-07-02T08:54:16.489+0000] {taskinstance.py:1308} INFO - Starting attempt 10 of 10
[2023-07-02T08:54:16.497+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): load_seed_data_once> on 2023-07-02 08:24:29.703225+00:00
[2023-07-02T08:54:16.501+0000] {standard_task_runner.py:57} INFO - Started process 282 to run task
[2023-07-02T08:54:16.504+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '1_init_once_seed_data', 'load_seed_data_once', 'manual__2023-07-02T08:24:29.703225+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/init.py', '--cfg-path', '/tmp/tmplmb__137']
[2023-07-02T08:54:16.505+0000] {standard_task_runner.py:85} INFO - Job 28: Subtask load_seed_data_once
[2023-07-02T08:54:16.545+0000] {task_command.py:410} INFO - Running <TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2023-07-02T08:24:29.703225+00:00 [running]> on host 7fea7a015b40
[2023-07-02T08:54:16.618+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='1_init_once_seed_data' AIRFLOW_CTX_TASK_ID='load_seed_data_once' AIRFLOW_CTX_EXECUTION_DATE='2023-07-02T08:24:29.703225+00:00' AIRFLOW_CTX_TRY_NUMBER='10' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-02T08:24:29.703225+00:00'
[2023-07-02T08:54:16.620+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-02T08:54:16.621+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt seed --profiles-dir .']
[2023-07-02T08:54:16.630+0000] {subprocess.py:86} INFO - Output:
[2023-07-02T08:54:18.128+0000] {subprocess.py:93} INFO - [0m08:54:18  Running with dbt=1.5.2
[2023-07-02T08:54:18.578+0000] {subprocess.py:93} INFO - [0m08:54:18  Registered adapter: snowflake=1.5.1
[2023-07-02T08:54:18.598+0000] {subprocess.py:93} INFO - [0m08:54:18  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2023-07-02T08:54:19.343+0000] {subprocess.py:93} INFO - [0m08:54:19  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-07-02T08:54:19.343+0000] {subprocess.py:93} INFO - There are 2 unused configuration paths:
[2023-07-02T08:54:19.344+0000] {subprocess.py:93} INFO - - models.dbt.analysis
[2023-07-02T08:54:19.344+0000] {subprocess.py:93} INFO - - models.dbt.transform
[2023-07-02T08:54:19.359+0000] {subprocess.py:93} INFO - [0m08:54:19  Found 5 models, 0 tests, 0 snapshots, 0 analyses, 438 macros, 0 operations, 3 seed files, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-07-02T08:54:19.361+0000] {subprocess.py:93} INFO - [0m08:54:19
[2023-07-02T08:54:20.766+0000] {subprocess.py:93} INFO - [0m08:54:20  Concurrency: 200 threads (target='dev')
[2023-07-02T08:54:20.768+0000] {subprocess.py:93} INFO - [0m08:54:20
[2023-07-02T08:54:20.795+0000] {subprocess.py:93} INFO - [0m08:54:20  1 of 3 START seed file public.bookings_1 ....................................... [RUN]
[2023-07-02T08:54:20.796+0000] {subprocess.py:93} INFO - [0m08:54:20  2 of 3 START seed file public.bookings_2 ....................................... [RUN]
[2023-07-02T08:54:20.797+0000] {subprocess.py:93} INFO - [0m08:54:20  3 of 3 START seed file public.customers ........................................ [RUN]
[2023-07-02T08:54:22.686+0000] {subprocess.py:93} INFO - [0m08:54:22  1 of 3 OK loaded seed file public.bookings_1 ................................... [[32mINSERT 7[0m in 1.88s]
[2023-07-02T08:54:22.687+0000] {subprocess.py:93} INFO - [0m08:54:22  2 of 3 OK loaded seed file public.bookings_2 ................................... [[32mINSERT 7[0m in 1.89s]
[2023-07-02T08:54:22.694+0000] {subprocess.py:93} INFO - [0m08:54:22  3 of 3 OK loaded seed file public.customers .................................... [[32mINSERT 2[0m in 1.89s]
[2023-07-02T08:54:22.726+0000] {subprocess.py:93} INFO - [0m08:54:22
[2023-07-02T08:54:22.727+0000] {subprocess.py:93} INFO - [0m08:54:22  Finished running 3 seeds in 0 hours 0 minutes and 3.36 seconds (3.36s).
[2023-07-02T08:54:22.739+0000] {subprocess.py:93} INFO - [0m08:54:22
[2023-07-02T08:54:22.740+0000] {subprocess.py:93} INFO - [0m08:54:22  [32mCompleted successfully[0m
[2023-07-02T08:54:22.741+0000] {subprocess.py:93} INFO - [0m08:54:22
[2023-07-02T08:54:22.742+0000] {subprocess.py:93} INFO - [0m08:54:22  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2023-07-02T08:54:23.464+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-02T08:54:23.486+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=1_init_once_seed_data, task_id=load_seed_data_once, execution_date=20230702T082429, start_date=20230702T085416, end_date=20230702T085423
[2023-07-02T08:54:23.518+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-02T08:54:23.530+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
